### PR TITLE
collect requests and limits for containers

### PR DESCRIPTION
**Pull request by zeari**
transferred from https://github.com/ManageIQ/manageiq/pull/15180

### CODE REVIEW

**Comment by simon3z** [comment]
@zeari can you fix this PR according to the failing tests? Thanks.

**Comment by simon3z** [comment]
@miq-bot assign Fryguy 

cc @Fryguy ready for merge

**Comment by zeari** [comment]
ping @Fryguy

File: db/migrate/20170528102055_add_limits_resources_to_containers.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class AddLimitsResourcesToContainers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :containers, :request_cpu,    :float  # cores
+    add_column :containers, :request_memory, :bigint # bytes
```
Comment (bdunne) [inline db/migrate/20170528102055_add_limits_resources_to_containers.rb]: Should the column_names have a `_cores` or `_bytes` suffix to eliminate future confusion?

Comment (zeari) [inline db/migrate/20170528102055_add_limits_resources_to_containers.rb]: @bdunne good idea

**Comment by zeari** [comment]
> Should the column_names have a _cores or _bytes suffix to eliminate future confusion?

@bdunne done

**Pull request merged**